mm/2s2h/Enhancements/Cutscenes/MiscInteractions/SkipMoonCrash.cpp

```diff
@@ -0,0 +1,43 @@
+#include <libultraship/bridge.h>
+#include <libultraship/luslog.h>
+#include "2s2h/GameInteractor/GameInteractor.h"
+#include "2s2h/ShipInit.hpp"
+
+extern "C" {
+#include "variables.h"
+}
+
+#define CVAR_NAME "gEnhancements.Cutscenes.SkipMiscInteractions"
+#define CVAR CVarGetInteger(CVAR_NAME, 0)
+
+// Didn't end up using these few cutscene values, but they are captured here as documentation in case they
+// are relevant in the future
+#define TERMINA_FIELD_MOON_CRASH_CS_ENTRANCE 0x54C0
+#define VOID_FIRE_WALL_CS_ENTRANCE 0x1C00
+#define VOID_FIRE_WALL_CS_INDEX 0xFFF8
```

Comment:
I think leaving them here is a smell, and might just lead to confusion as people glaze over the comment
```suggestion
#define TERMINA_FIELD_MOON_CRASH_CS_ENTRANCE 0x54C0
```